app/src/lib/notifications/show-notification.ts

```diff
@@ -0,0 +1,39 @@
+import { focusWindow } from '../../ui/main-process-proxy'
+import { supportsNotifications } from 'desktop-notifications'
+import { showNotification as invokeShowNotification } from '../../ui/main-process-proxy'
+import { notificationCallbacks } from './notification-handler'
+import { DesktopAliveEvent } from '../stores/alive-store'
+
+interface IShowNotificationOptions {
+  title: string
+  body: string
+  userInfo?: DesktopAliveEvent
+  onClick: () => void
+}
+
+/**
+ * Shows a notification with a title, a body, and a function to handle when the
+ * user clicks on the notification.
+ */
+export async function showNotification(options: IShowNotificationOptions) {
+  if (!supportsNotifications()) {
+    const notification = new Notification(options.title, {
```

Comment:
I am probably forgetting something (or lack of sleep is making it not click..) but, this reads as "If we don't support notifications, make a new notification".  Is this a dual support of a kind that is notified by renderer vs main process?

Reply:
Yes, this would be HTML5 notification API. I'll write a comment to make it less confusing 😳  Sorry about that!

Reply:
Fixed in 5d62f75c6dc028cea5bc6ce7cf2aaa2dd99717f1